drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -227,12 +227,21 @@ function DragAndDropTemplates(configuration) {
     };
 
     var feedbackTemplate = function(ctx) {
-        var feedback_display = ctx.feedback_html ? 'block' : 'none';
         var properties = { attributes: { 'aria-live': 'polite' } };
+        var messages = ctx.overall_feedback_messages || [];
+        var feedback_display = messages.length > 0 ? 'block' : 'none';
+        var feedback_messages = messages.map(function(message) {
+            var selector = "p.message";
+            if (message.message_class) {
+                selector += "."+message.message_class;
+            }
+            return h(selector, {innerHTML: message.message}, []);
```

Comment:
@cahrens: As you noted, we were terribly sloppy in the past and basically everything allowed HTML (and thus was vulnerable to XSS).  Many people used this as a feature in certain areas.  Now, we simply need to be more intentional and determine whether HTML is an acceptable input in certain places.  Documentation should probably be updated to reflect the intentional decision where HTML is accepted.

Additionally, where HTML is accepted, I guess we could further determine if limited HTML is accepted or any HTML.  @nedbat?